page-searchbox_multi.php

```diff
@@ -0,0 +1,143 @@
+      }
+    </style>
+  </head>
+  <body>
+    <main style="display:flex;max-height:90px;width:450px;gap:.5rem;">
+        <img src="https://babel.hathitrust.org/common/firebird/dist/hathitrust-icon-orange.svg" alt="HathiTrust widget search" />
+        <div style="width:100%;">
+          <h1>Search HathiTrust Digital Library Catalog</h1>
+          <div>
+            <form
+              method="get"
+              action="https://catalog.hathitrust.org/Search/Home"
+              name="HT_CatalogSearchForm_01"
+              class="search"
+              id="widget-catalog-multi" 
+              target="_blank"
+            >
+              <div >
+                <input type="hidden" name="checkspelling" value="true" />
+                <input type="hidden" name="type" value="all" />
+                <label for="lookfor" class="offscreen">Search Terms</label>
+                <div style="display:flex;gap:5px;">
+                  <input
+                    class="searchterms"
+                    type="text"
+                    name="lookfor"
+                    value=""
+                    id="lookfor"
+                   style="flex-grow:2" 
+                  />
+                  <button type="submit" name="submit" id="searchSubmit" >Find</button>
+                </div>
+              </div>
+              <div style="display:flex;justify-content:space-between;gap:10px;margin-block:3px">
+                <label class="offscreen" for="searchtype">Field Name</label>
+                <select style="flex-grow:2" id="searchtype" name="type" >
+                  <option value="all">All Fields</option>
+                  <option value="title">Title</option>
+                  <option value="author">Author</option>
+                  <option value="subject">Subject</option>
+                  <!--<option value="hlb">Academic Discipline</option>-->
+                  <!--<option value="callnumber">Call Number / in progress</option>-->
+                  <option value="isn">ISBN/ISSN</option>
+                  <option value="publisher">Publisher</option>
+                  <option value="series">Series Title</option>
+                  <option value="year">Year of Publication</option>
+                  <!-- <option value="tag">Tag</option> -->
+                </select>
+                  <div style="display:flex;align-items:center;gap:3px;">
+                  <input type="hidden" name="sethtftonly" value="true" />
+                  <input
+                    type="checkbox"
+                    name="htftonly"
+                    value="true"
+                    id="fullonly"
+                    checked="yes"
+                    
+                  /><label
+                    for="fullonly"
+                    style="font-size:75%;"
+                    
+                    >Full view only</label
+                  >
+                </div>
+              </div>
+              </div>
+              <div >

```

Comment:
The preview seems to think there is an extra `div` in here. I think tightening up the spacing could make this a bit easier to scan, regardless of whether the `div`s are balanced.

Also, going one line up, is `checked="yes"` valid XHTML transitional?

Reply:
yes, there was an extra div, but it was near the end. It's gone now, so that should be cleared up. And thanks for asking about `checked="yes"`, and I changed all the checkbox inputs to `checked="checked"`. These widgets are _old_.